### DISCUSSION TITLE
【灌水】我来绳网只为说一件事，

### DISCUSSION THREAD

**Discussion by WHKKa** [share121/inter-knot#2291] [其他]
我是傻逼。
哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈<img src="https://upload-bbs.miyoushe.com/upload/2022/12/21/6bf4ba03d77560e797085eb691f40b91_4095837989954111024.png" alt="时光交响之章表情包14" width="75" >

**Comment by KUYUE404** [comment]
哎嘿<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by HKFDBHZN** [comment]
这个**我接了
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
（bushi

**Comment by LuAn-afl** [comment]
大可不必

**Comment by LuAn-afl** [comment]
大可不必

**Comment by liu20040515** [comment]
我们知道了

**Comment by Momsaysnamesthataretoolongmakefoolslook** [comment]
温迪，斯哈斯哈
